packages/sanity/src/core/form/inputs/InvalidValueInput/UntypedValueInput.tsx

```diff
@@ -69,31 +86,32 @@ export function UntypedValueInput({validTypes, value, onChange}: UntypedValueInp
   const isSingleValidType = validTypes.length === 1
   const isHoistedType = schema.has(validTypes[0])
 
+  const {t} = useTranslation()
+
   return (
     <Alert
       status="warning"
       title={
         <>
-          Property value missing <code>_type</code>
+          <Translate t={t} i18nKey="inputs.untyped-value.title" />
```

Comment:
not a blocking comment but i think this can be replaced with just a `t()` instead of the translate component. it's somewhat preferred because it's a bit more light weight but mostly negligible

Reply:
I initially thought the same, but as the translated string is using HTML tags (`<code>`), it cannot - it needs to use the `Translate` component. A bit unfortunate, but this is the way react-i18next works.